### ISSUE TITLE
Get a framework

### ISSUE THREAD

**Issue by matt-trinh** [mfefer/mfefer.github.io#1]
Probably some time before June 12

**Comment by mfefer** [comment]
Is by May 24 fine? That's my first conference. If not June 12 is also very gucci

**Comment by matt-trinh** [comment]
yeah that works, just wanted to bump this, i'll formalize some stuff later
